lib/pynput/keyboard/_xorg.py

```diff
@@ -154,7 +165,13 @@ class Key(enum.Enum):
     f19 = KeyCode._from_symbol('F19')
     f20 = KeyCode._from_symbol('F20')
     home = KeyCode._from_symbol('Home')
+    hyper = KeyCode._from_symbol('Hyper_L')
```

Comment:
Many of these additions are specific to the _X_ backend, and I fear that adding them might lead users of this library to write non-portable code that works on their system but on theirs users' systems. What do you think about adding a prefix, like `xorg_`, to these additions?

Reply:
Solved in https://github.com/moses-palmer/pynput/pull/401/commits/7ce556e5f0dd6161c2ab7a0457e0ac2e319bbe01